### PR TITLE
Private/mmeeks/bgsave2

### PR DESCRIPTION
**Pull request by mmeeks**
* Resolves: # 
* Target version: master 

### TODO

- [ ] ...

### Checklist

- [ ] Code is properly formatted
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

**Comment by caolanm** [comment]
lots of failures along the lines of the "COOL 24.04 unit tests w/o cypress" seen in ci and locally since this one.

**Comment by mmeeks** [comment]
Ah drat - is it me busting CI somehow ... hmm.

File: wsd/DocumentBroker.cpp [hunk 1]
```diff
@@ -2673,23 +2673,15 @@ bool DocumentBroker::sendUnoSave(const std::shared_ptr<ClientSession>& session,
         // We do not want save to terminate editing mode if we are in edit mode now.
         //TODO: Perhaps we want to terminate if forced by the user,
         // otherwise autosave doesn't terminate?
-        oss << "\"DontTerminateEdit\":"
```
Comment (caolanm) [inline wsd/DocumentBroker.cpp:2676]: I think it is the innocent looking changes, they introduce extra spaces that something doesn't expect. At least in the first case of unit-wopi-lock I look at

kit-3852552-3852552 2024-04-16 09:50:57.973959 +0100 [ kitbroker_002 ] ERR  ToMaster-00b: Exception while handling [save autosave= {"DontTerminateEdit" : { "type":"boolean", "value":true" },"DontSaveIfUnmodified" : { "type":"boolean", "value":true" }}]: <unspecified file>(1): expected '}' or ','| common/Session.cpp:306

File: wsd/DocumentBroker.cpp [hunk 2]
```diff
@@ -2673,23 +2673,15 @@ bool DocumentBroker::sendUnoSave(const std::shared_ptr<ClientSession>& session,
...
-               "{"
-               "\"type\":\"boolean\","
-               "\"value\":true"
-               "}";
+        oss << "\"DontTerminateEdit\" : { \"type\":\"boolean\", \"value\":true\" }";
```
Comment (caolanm) [inline wsd/DocumentBroker.cpp:2676]: no, I was close, but its a stray \", https://github.com/CollaboraOnline/online/pull/8776 for that at least

**Comment by mmeeks** [comment]
Thanks for nailing that!

**Pull request merged**